lib/view/search_bar.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+/// SearchBar:
+///   - a widget representing a search bar.
+class SearchBar extends StatefulWidget {
+  final void Function(String)? onChanged;
+
+  const SearchBar({super.key, this.onChanged});
+
+  @override
+  State<SearchBar> createState() => _SearchBarState();
+}
+
+class _SearchBarState extends State<SearchBar> {
+  final TextEditingController editingController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+        width: 150,
```

Comment:
```suggestion
        width: 160,
```
Like I said, increments of 4 are usually best for consistency.